dns-records.tf

```diff
@@ -11,15 +12,24 @@ resource "azurerm_dns_a_record" "cert-ci-jenkins-io" {
   tags = local.default_tags
 }
 
-# CNAME record for artifact-caching-proxy on Azure
-resource "azurerm_dns_cname_record" "target" {
-  name                = "repo.azure"
+### CNAME records
+# CNAME records targetting the public-nginx on publick8s cluster
+resource "azurerm_dns_cname_record" "target_public_publick8s" {
+  # Map of records and corresponding purposes
+  for_each = {
+    "repo.azure" = "artifact-caching-proxy"
+    "weekly.ci"  = "Jenkins Weekly"
```

Comment:
```suggestion
    "weekly.ci"  = "Jenkins Weekly Public Demo Controller"
```

NIT: Jenkins Weekly is confusing (is it the weekly release? The weekly demonstrator ? The Weekly instance ?)